### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by kmike**
I'm working on a small cleanup for the documentation. The goal is to make API reference look ok, there are some issues in current docs - many important methods are not documented, docs for classes are mixed with module docstring.

### CODE REVIEW

**Comment by kmike** [comment]
The PR is ready for review.

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     description="Zyte's Page Object pattern for web scraping",
     long_description=long_description,
     long_description_content_type="text/x-rst",
-    author="Scrapinghub",
+    author="Zyte Group Ltd",
     author_email="info@scrapinghub.com",
```
Comment (BurnzZ) [inline setup.py:14]: We could update the email as well to `info@zyte.com` to be in line with the doc changes.

File: web_poet/page_inputs/client.py [hunk 2]
```diff
@@ -191,16 +184,16 @@ async def batch_execute(
         responses despite any possible failures. This can be done by setting
         ``True`` to the ``return_exceptions`` parameter.
 
-        Like :meth:`~.HttpClient.execute`, :class:`web_poet.exceptions.http.HttpResponseError`
+        Like :meth:`~.HttpClient.execute`, :class:`~.HttpResponseError`
         will be raised for responses with status codes in the ``400-5xx`` range.
```
Comment (BurnzZ) [inline web_poet/page_inputs/client.py:188]: I noticed that "\`\`400-5xx\`\`" and "\`\`100-3xx\`\`" are mentioned simply as "400-5xx" and "100-3xx" in some parts of the code. Let's update this to be consistent with a particular format. I'm leaning towards formatting it with "\`\`" to make them more prominent in the docs. We could also make it **bold**.

Do you have some preference on this?

**Review by BurnzZ — APPROVED** [review]
I only have some minor changes noted down but aside from that looking good @kmike !

File: web_poet/page_inputs/client.py (hunk 2 above)
Comment (kmike) [reply web_poet/page_inputs/client.py:188]: Changed them to the existing style in https://github.com/scrapinghub/web-poet/pull/50/commits/b5e869049cf15327cc0ecc9bf0ed5188b5101d9f, thanks @BurnzZ!

File: setup.py (hunk 1 above)
Comment (kmike) [reply setup.py:14]: @Gallaecio could you please advice us here? When doing the rebranding, what was the email you were using?

Comment (Gallaecio) [reply setup.py:14]: For open source projects I use opensource@zyte.com.

File: web_poet/page_inputs/client.py [hunk 3]
```diff
@@ -33,20 +23,19 @@
...
+    :mod:`contextvar <contextvars>`, or a function passed via
+    ``request_downloader`` argument of the :meth:`~.HttpClient.__init__` method.
 
-    In any case, this doesn't contain any implementation about how to execute
-    any requests fed into it. When setting that up, make sure that the downloader
-    implementation returns a :class:`~.HttpResponse` instance.
+    Either way, this function should be an ``async def`` function which receives
+    :class:`~.HttpRequest` instance, and either returns :class:`~.HttpResponse`
+    instance, or raises a subclass of :class:`~.HttpError`. You can read
+    more in the :ref:`advanced-downloader-impl` documentation.
```
Comment (Gallaecio) [inline web_poet/page_inputs/client.py:38]: X instance → an X instance

```suggestion
    Either way, this function should be an ``async def`` function which 
    receives an  :class:`~.HttpRequest` instance, and either returns a
    :class:`~.HttpResponse` instance, or raises a subclass of 
    :class:`~.HttpError`. You can read more in the 
    :ref:`advanced-downloader-impl` documentation.
```

File: web_poet/page_inputs/client.py [hunk 4]
```diff
@@ -83,28 +72,27 @@ async def request(
...
-        A :class:`web_poet.exceptions.http.HttpRequestError` will be raised on
-        cases like *connection errors*, *connection and read timeouts*, etc.
+        :class:`~.HttpRequestError` is raised for
+        *connection errors*, *connection and read timeouts*, etc.
 
-        A :class:`~.HttpResponse` instance should then be returned for successful
-        responses in the 100-3xx status code range. Otherwise, an exception of
-        type :class:`web_poet.exceptions.http.HttpResponseError` will be raised.
+        :class:`~.HttpResponse` instance is returned for successful
+        responses in the ``100-3xx`` status code range.
```
Comment (Gallaecio) [inline web_poet/page_inputs/client.py:82]:
```suggestion
        An :class:`~.HttpResponse` instance is returned for successful
        responses in the ``100-3xx`` status code range.
```

**Review by Gallaecio — APPROVED** [review]

**Comment by kmike** [comment]
Thanks @BurnzZ and @Gallaecio!

**Pull request merged**